apis/src/pwa_cache.rs

```diff
@@ -0,0 +1,27 @@
+#[cfg(feature = "ssr")]
+use actix_web::{get, HttpResponse, Responder, web};
+use walkdir::WalkDir;
+
+#[cfg(feature = "ssr")]
+#[get("/pwa-cache")]
+pub async fn pwa_cache(site_root: web::Data<String>) -> impl Responder {
```

Comment:
This might deserve some memoization. But, for now, it should be fine as the tree of assets is quite small and that would be a too-early optimisation.
WDYT?